lms/services/dashboard.py

```diff
@@ -172,12 +176,43 @@ def get_request_admin_organizations(self, request) -> list[Organization]:
             else request.identity.userid,
         )
 
+    def get_assignment_roster(
+        self, assignment: Assignment, h_userids: list[str] | None = None
+    ) -> Select[tuple[LMSUser]]:
+        rosters_enabled = (
+            assignment.course
+            and assignment.course.application_instance.settings.get(
+                "hypothesis", "dashboard_rosters"
+            )
+        )
+        if rosters_enabled and self._roster_service.assignment_roster_exists(
+            assignment
+        ):
+            # If rostering is enabled and we do have the data, use it
+            query = self._roster_service.get_assignment_roster(
+                assignment, h_userids=h_userids
+            )
+
+        else:
+            # Always fallback to fetch users that have launched the assignment at some point
+            query = self._user_service.get_users_for_assignment(
+                role_scope=RoleScope.COURSE,
+                role_type=RoleType.LEARNER,
+                assignment_id=assignment.id,
+                h_userids=h_userids,
+            )
+
+        # Always return the results, no matter the source, sorted
+        return query.order_by(LMSUser.display_name, LMSUser.id)
```

Comment:
Isn't this comment saying the opposite that is reflected in code? 🤔 

* What I understand from the comment: We are not sorting so that other can decide how to sort.
* The code: Let's sort by display name and id 😅

Reply:
Sorry, I reckon the comment on the code is right but my comment on the PR is in the wrong place, this comment (again, the PR one) should be on the service (UserService) where we have removed the sorting.

Reply:
> my comment on the PR is in the wrong place, this comment (again, the PR one) should be on the service (UserService) where we have removed the sorting.

Ok, that makes sense 👍🏼